### DISCUSSION TITLE
how to use GoLand to debug juicefs code

### DISCUSSION THREAD

**Discussion by mupengfeilove** [juicedata/juicefs#4298] [General]
how to use GoLand to debug juicefs code

**Comment by zhijian-pro** [comment]
1. use `juicefs format` first
2. <img width="2417" alt="image" src="https://github.com/juicedata/juicefs/assets/31313340/9880015c-8299-449e-9bdc-6fc7d9721065">
3. <img width="1157" alt="image" src="https://github.com/juicedata/juicefs/assets/31313340/202aa3fe-d502-49cc-a59e-991ffc8327f1">
4. <img width="1560" alt="image" src="https://github.com/juicedata/juicefs/assets/31313340/2ca4f542-f4a8-4a9d-be99-4e1d8bfc896f">
Do not add `-d` arguments
